website/package.json

```
@@ -12,12 +12,13 @@
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
-    "typecheck": "tsc"
+    "tsc": "tsc"
```

I would revert that, no need to change it IMO, it's coming from the template of docusaurus, we also use the same name of script in LL, it's clear enough and with pnpm you can already use tsc by calling `pnpm tsc`
```suggestion
    "typecheck": "tsc"
```

---

why do you prefer the longer name, won't it be easier for a dev to type `pnpm tsc`?

---

It's not really about preferring the longer name, but more that the template comes with this command and `typecheck` is pretty standard even in LL we use the same name, also tsc and typecheck is not really the same as tsc is also a build tool

---

not the same? but the value of `tsc` is `tsc`?